libs/common/package.json

```diff
@@ -28,6 +28,7 @@
     "db:migrate": "npm run typeorm -- migration:run -d src/sql-db-connection/data-source.ts",
     "db:migrate:down": "npm run typeorm -- migration:revert -d src/sql-db-connection/data-source.ts",
     "db:reset": "npm run db:drop-all && npm run db:migrate",
```

Comment:
I am not sure how this has passed the PR, but we simply cannot have NPM commands which allow DB to be dropped and displayed as this will introduce a vulnerability.

Reply:
This is a command that can only run locally. I believe that the user we set up for github QAT pipeline for example does not have sufficient permissions to drop the db. Where are you concerned that a vulnerability could be introduced by this npm command?

Reply:
A script injected into the system which can trigger this command (highly unlikely) but still should not have such commands.

Reply:
I am fairly confident that running that command even directly would not drop the db due to the user permissions. For example, in the `test.yml`, we have to use `npm run db:migrate` directly rather than running `npm run db:reset` as the latter will fail to execute due to the `DTFS` user not having sufficient permissions